src/app/src/main/kotlin/org/vaccineimpact/api/app/controllers/ResponsibilityController.kt

```diff
@@ -14,17 +15,18 @@ import org.vaccineimpact.api.models.TouchstoneVersion
 
 class ResponsibilityController(
```

Comment:
Maybe rename this to `GroupResponsibilityController` now that there are responsibilities on another controller.